deps/ext.jl.bak

```diff
@@ -0,0 +1,6 @@
+# autogenerated file, do not edit
```

Comment:
Don't check this file in.

Reply:
oh, I guess the gitignore needs some tweaking